examples/tv-app/android/java/MediaPlaybackManager.h

```diff
@@ -120,5 +121,6 @@ class MediaPlaybackManager : public MediaPlaybackDelegate
     HandleMediaRequest(MediaPlaybackRequest mediaPlaybackRequest, uint64_t deltaPositionMilliseconds);
 
     // TODO: set this based upon meta data from app
-    uint32_t mDynamicEndpointFeatureMap = 3;
+    static constexpr uint32_t kEndpointFeatureMap = 3;
```

Comment:
similar here: feature map should explain what features are enabled here - ideally typesafe code, failing that at least comments.

Reply:
Created an issue
https://github.com/project-chip/connectedhomeip/issues/31991